lib/taskana-core/src/main/java/pro/taskana/impl/ClassificationQueryImpl.java

```diff
@@ -587,13 +467,13 @@ public String toString() {
         builder.append(Arrays.toString(domain));
         builder.append(", validInDomain=");
         builder.append(validInDomain);
-        builder.append(", created=");
-        builder.append(Arrays.toString(created));
+        builder.append(", createdIn=");
+        builder.append(Arrays.toString(createdIn));
         builder.append(", nameIn=");
         builder.append(Arrays.toString(nameIn));
         builder.append(", nameLike=");
         builder.append(Arrays.toString(nameLike));
-        builder.append(", descriptionLike=");
+        builder.append(", description=");
```

Comment:
There is "DescriptionLike" as filter at the QueryMapper. So the naming was correct i guess.

Reply:
ok, I renamed the 'description' variable, although this has no effect on the correct method name .